core/file_api.php

```diff
@@ -69,8 +69,47 @@ function file_attach_files( $p_bug_id, $p_files, $p_bugnote_id = 0 ) {
 	}
 
 	$t_file_infos = array();
+	$t_zip_attachments = config_get( 'zip_attachments' );
+	$t_zip_go = false;
```

Comment:
This extra variable is not necessary. Just use $t_zip_attachments, as I suggested in my 1st review

Reply:
Adjusted accordingly